<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>buyGoods</class>
 <widget class="QDialog" name="buyGoods">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>640</width>
     <height>400</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>已有商品</string>
    </attribute>
    <widget class="QPushButton" name="enterButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>300</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>251</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="goodsNumLayout_2">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>商品编号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="goodsNum">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="goodsQuantityLayout_2">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>数   量：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="goodsQuantity">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="cancelButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>300</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>新商品</string>
    </attribute>
    <widget class="QPushButton" name="newEnterButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>320</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="newCancelButton">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>320</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="discountPercent">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>260</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>60</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>21</height>
      </size>
     </property>
     <property name="text">
      <string>折扣率：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>300</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>144</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>生产厂家：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="produce">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>300</y>
       <width>203</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>21</y>
       <width>351</width>
       <height>40</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="goodsNumLayout_3">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>商品编号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="goodsNum2"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>73</y>
       <width>351</width>
       <height>40</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="goodsQuantityLayout_3">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>数    量：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="goodsQuantity2"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>125</y>
       <width>351</width>
       <height>40</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>商品名称：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="goodsName"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>177</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>销售价格：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="salePrice"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>220</y>
       <width>60</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>21</height>
      </size>
     </property>
     <property name="text">
      <string>积分率：</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="scorePercent">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>220</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>label_9</zorder>
    <zorder>scorePercent</zorder>
    <zorder>newEnterButton</zorder>
    <zorder>newCancelButton</zorder>
    <zorder>discountPercent</zorder>
    <zorder>label_10</zorder>
    <zorder>label_11</zorder>
    <zorder>produce</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
